en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -264,31 +264,23 @@ From the above two figures, notice the directory generated after building the __
 
 #### Native
 
-> **Note**: this part has not been updated to the latest version yet, please refer to the actual build result.
+The release package generated by Cocos Creator 2.4.3 after building the __Windows__ platform is as follows:
 
-The directory generated by Cocos Creator 2.4.3 after building the __Windows__ platform is as follows:
-
-![image](window-v243.png)
-
-The directory generated by Cocos Creator 3.0 after building the __Windows__ platform is as follows:
-
-![image](window-v3.png)
-
-As you can see from the above two figures, there is a big difference between v2.4.3 and v3.0 in the directory generated after building the __Windows__ platform.
-
-Since the underlying C++ code generated after building on each native platform (e.g. Android, Windows) is completely consistent. Cocos Creator 3.0 extracts the underlying C++ code that was stored in the `jsb-link/frameworks/runtime-src/Classes` directory in v2.4.3 and placed it in a shared `common-link` folder. This way, when building the native platform, if the `common-link` folder is detected to already exist, this part will no longer be processed, to speed up the build.
+![image](v243-windows.png)
 
-The `Windows` folder is used by v3.0 to store the native-related content for the current built (or the `android` folder if the build is for Android).
+The release package generated by Cocos Creator 3.0 after building the __Windows__ platform is as follows:
```

Comment:
```suggestion
The release package generated by Cocos Creator v3.0 after building the __Windows__ platform is as follows:
```

Reply:
It should be "Cocos Creator 3.0" or "v3.0",we have previously discussed and determined in this pr: https://github.com/cocos-creator/creator-docs/pull/1401